examples/tracer-web/examples/xml-http-request/index.js

```diff
@@ -0,0 +1,59 @@
+import { ConsoleSpanExporter, SimpleSpanProcessor } from '@opentelemetry/tracing';
+import { WebTracer } from '@opentelemetry/web';
+import { XMLHttpRequestPlugin } from '@opentelemetry/plugin-xml-http-request';
+import { ZoneScopeManager } from '@opentelemetry/scope-zone';
+import { CollectorExporter } from '@opentelemetry/exporter-collector';
+import { B3Format } from '@opentelemetry/core';
+
+const webTracerWithZone = new WebTracer({
+  httpTextFormat: new B3Format(),
+  scopeManager: new ZoneScopeManager(),
+  plugins: [
+    new XMLHttpRequestPlugin({
+      ignoreUrls: ['http://localhost:8090/sockjs-node/info'],
+      propagateTraceHeaderCorsUrls: [
+        'http://localhost:8090'
```

Comment:
Is `'http://localhost:8090'` a CORS URL in this example? Or is it the origin that the example is served from?

If it's the origin the example is served from, can we remove it from the list?

Reply:
cors, fixed